### PR TITLE
Use denite in contextMenu

### PR DESCRIPTION
**Pull request by ficoos**
When using denite for the context menu, the actions should be opened
in denite.

### CODE REVIEW

File: rplugin/python3/denite/source/contextMenu.py [hunk 1]
```diff
@@ -10,10 +18,17 @@ def __init__(self, vim):
         self.kind = 'command'
 
     def convert_to_candidate(self, item):
+        if item in DeniteOverrides:
+            cmd = 'call denite#start({})'.format(
+                repr([{"name": DeniteOverrides[item],
```
Comment (autozimu) [inline rplugin/python3/denite/source/contextMenu.py]: Is this `repr` necessary here?

Comment (ficoos) [reply rplugin/python3/denite/source/contextMenu.py]: For lists `str()` calls `repr()` on the underlying objects so the effect is identical. But the *intent* is to use `repr()`. If it wasn't a list, maybe a string for example, `repr()` would have been required.

This documents the fact that we *want* the `repr()` of the object and not the `str()` and ignores the fact that for lists coincidentally `str() == repr()`.

Comment (autozimu) [reply rplugin/python3/denite/source/contextMenu.py]: My point here is, `repr` is not necessary, nor `str`, as the goal is formatting one string.

How about this?
```python
cmd = 'call denite#start(["name": {}, "args": []])'.format(DeniteOverrides[item])
```

Comment (ficoos) [reply rplugin/python3/denite/source/contextMenu.py]: Oh, I didn't get that this is what you meant. I guess you are right.

**Comment by autozimu** [comment]
Thanks.

**Pull request merged**